recipes/newrelic/apm/dotNet/debian-systemd-nginx.yml

```diff
@@ -0,0 +1,284 @@
+# Visit our schema definition for additional information on this file format
+# https://github.com/newrelic/open-install-library/blob/main/docs/recipe-spec/recipe-spec.md#schema-definition
+
+name: dotnet-agent-installer
+displayName: .NET Agent Installer
+description: New Relic install recipe for instrumenting .NET applications on Linux
+repository: https://github.com/newrelic/newrelic-dotnet-agent
+
+installTargets:
+  - type: application
+    os: linux
+    platform: "debian"
+  - type: application
+    os: linux
+    platform: "ubuntu"
+
+keywords:
+  - Apm
```

Comment:
@nr-ahemsath  please remove `Apm` keyword for now, we'll add it back later to have this recipe included in guided install once the detection mechanism enhancement is ready in https://newrelic.atlassian.net/browse/VIRTUOSO-786

Reply:
Done, thanks.